src/controller.coffee

```diff
@@ -4287,15 +4316,31 @@ Editor::addLineNumberForLine = (line) ->
     @lineNumberTags[line] = lineDiv
 
   lineDiv.className = 'droplet-gutter-line'
-  if @gutterDecorations[line]
-    lineDiv.className += ' ' + @gutterDecorations[line].join(' ')
-  lineDiv.style.top = "#{treeView.bounds[line].y - @scrollOffsets.main.y}px"
+
+  # Add annotation mouseover text
+  # and graphics
+  if @annotations[line]?
+    lineDiv.className += ' droplet_' + getMostSevereAnnotationType(@annotations[line])
+    lineDiv.style.backgroundPosition = "2px #{treeView.distanceToBase[line].above - @view.opts.textHeight - @fontAscent}px"
```

Comment:
is this taking precedence over the earlier CSS? If you find a way to make it work with "2px center" in the CSS, you may not need to do this.